src/engine/box/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+# target_sources(app PRIVATE 
+#     box.cpp
+#     rectangle.cpp
+# )
```

Comment:
is this file supposed to exist at all? :laughing:

Reply:
It is not required, and currentlmy commented because it refers to deleted file (see one of the previous resolved convo) but I decided to left it anyway if it is required at some point, and also when we will "clean" the includes of the template file.

But as for right now yes it is a dead file, so I can remove it if you wish so

Reply:
It's fine as is don't worry, there will be more cleaning to do anyway!